9/run-demo-load-test.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+docker build -t vegeta-loadtest -f ./ops/Vegeta/Dockerfile ./ &&
+  docker run -it --rm vegeta-loadtest
```

Comment:
These load test scripts should probably be revised. The vegeta script in this solution is targeting one of the older controllers (hello world). Probably best to remove the load test altogether.

Also, the `/imgs` files are for a different solution and should ideally be removed too.